server/sonar-web/src/main/js/components/controls/HomePageSelect.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { connect } from 'react-redux';
+import Tooltip from './Tooltip';
+import HomeIcon from '../icons-components/HomeIcon';
+import { CurrentUser, isLoggedIn, HomePage, isSameHomePage } from '../../app/types';
+import { translate } from '../../helpers/l10n';
+import { getCurrentUser } from '../../store/rootReducer';
+import { setHomePage } from '../../store/users/actions';
+
+interface StateProps {
+  currentUser: CurrentUser;
+}
+
+interface DispatchProps {
+  setHomePage: (homepage: HomePage) => void;
+}
+
+interface Props extends StateProps, DispatchProps {
+  className?: string;
+  currentPage: HomePage;
+}
+
+class HomePageSelect extends React.PureComponent<Props> {
+  handleClick = (event: React.SyntheticEvent<HTMLAnchorElement>) => {
+    event.preventDefault();
+    event.currentTarget.blur();
+    this.props.setHomePage(this.props.currentPage);
+  };
+
+  render() {
+    const { currentPage, currentUser } = this.props;
+
+    if (!isLoggedIn(currentUser)) {
+      return null;
+    }
+
+    const { homepage } = currentUser;
+    const checked = homepage !== undefined && isSameHomePage(homepage, currentPage);
+    const tooltip = checked ? translate('homepage.current') : translate('homepage.check');
+
+    return (
+      <Tooltip overlay={tooltip}>
```

Comment:
The tooltip should be placed on the side, because right now it's not correctly placed.